mitodl/micromasters
# Add coupon invoice table

#### What are the relevant tickets?
Fixes #2489 

#### What's this PR do?
Adds a `CouponInvoice` table with a one to many relationship to `Coupon`, as well as an invoice number and a description.

#### How should this be manually tested?
Create a coupon invoice through the Django admin and verify that the `CouponInvoiceAudit` table was updated
